services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        movies = Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids and not actors_ids:
+        movies = Movie.objects.filter(genres__id__in=genres_ids)
+
+    if not genres_ids and actors_ids:
+        movies = Movie.objects.filter(actors__id__in=actors_ids)
```

Comment:
You could simplify this part of the code to have only two if conditions. If `genres_ids` are present, filter the queryset in the `movies` variable by them. If `actors_ids` are present, filter the queryset in the `movies` variable by them. There is no need to define the new queryset each time.